thirdeye/thirdeye-frontend/app/utils/utils.js

```diff
@@ -32,7 +31,7 @@ export function checkStatus(response, mode = 'get', recoverBlank = false) {
  * Formatter for the human-readable floating point numbers numbers
  */
 export function humanizeFloat(f) {
-  if (isNone(f) || Number.isNaN(f)) { return '-'; }
+  if (typeof f !== 'number') { return '-'; }
```

Comment:
Instead of having a check for whether f is a number in every single util function, it might be more concise to just check for f's type first and if it's a number do these other functions or if not then you only have to write the check once. Not worth a refactor or anything, just a thought for future purposes where you find yourself writing a the same logic repeatedly